File: preview/understand/field_value/fetch-default/fetch-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Preview.Understand.Service.FieldType;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var fieldValue = FieldValueResource.Fetch(
            pathServiceSid: "UAaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
            pathFieldTypeSid: "UBaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
            pathSid: "UCaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
        );

        Console.WriteLine(fieldValue.AccountSid);
    }
}

File: preview/understand/field_value/fetch-default/fetch-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

field_value = @client.preview
                     .understand
                     .assistants('UAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                     .field_types('UBXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                     .field_values('UCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                     .fetch

puts field_value.date_created
